### ISSUE TITLE
[a11y]: [WebTerminal] [Level-1 Violation] The ARIA attributes "aria-label" are not valid for the element

### ISSUE THREAD

**Issue by yawenchien** [carbon-design-system/ibm-products#7950]
### Package

@carbon/ibm-products

### Browser

Edge

### Operating System

MacOS

### Package version

v2.70.1

### Automated testing tool and ruleset

IBM Equal Access Accessibility Checker 4.0.6

### Description

<img width="701" height="175" alt="Image" src="https://github.com/user-attachments/assets/7cf7e530-4185-484a-9b7e-9bdb76993e8f" />

**Violation**
The ARIA attributes "aria-label" are not valid for the element <header> with implicit ARIA role "generic"
ARIA attributes should be valid for the element and ARIA role to which they are assigned

**Why is this important?**
When elements are assigned an ARIA role, ARIA specifies which attributes are valid for that role. If invalid ARIA attributes are used, assistive technologies may not be able to interpret and operate the element accurately for users.

**Element location**
```
<header
    class="c4p--web-terminal__bar"
    aria-label="Web terminal header">
```

**What to do**
- Remove ARIA state or property attributes that are not valid for the element role
- Or, change to a role that supports the intended behavior of the element, using the ARIA specification as a guide

In the following example, the element references only valid ARIA attributes for an element with the role of `"tree"`, a global `aria-labelledby` property, and an `aria-expanded` state:
```
<ul id="tree1" class="tree root-level" role="tree" aria-labelledby="label_1">
    
<li id="fruits" class="tree-parent" role="treeitem" tabindex="0" aria-expanded="true">
    <span>Fruits</span>
```

Rule ID: `aria_attribute_valid`
Reason ID: `Fail_invalid_implicit_role_attr`

About this requirement
- [ARIA specification - States and Properties](https://www.w3.org/TR/wai-aria-1.2/#introstates)
- [ACT rule - ARIA state or property is permitted](https://www.w3.org/WAI/standards-guidelines/act/rules/5c01ea/)
- [ACT rule - Element marked as decorative is not exposed](https://www.w3.org/WAI/standards-guidelines/act/rules/46ca7f/)

**Who does this affect?**
People using a screen reader, including blind, low vision, and neurodivergent people
People with dexterity impairments using voice control

### Reproduction/example

https://carbon-for-ibm-products.netlify.app/?path=/docs/patterns-prebuilt-patterns-webterminal--overview

### Steps to reproduce

`WebTerminal` without actions
<img width="701" height="175" alt="Image" src="https://github.com/user-attachments/assets/7cf7e530-4185-484a-9b7e-9bdb76993e8f" />

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/ibm-products/blob/main/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/ibm-products/issues) for duplicate problems
